src/Resources/config/listeners.yml

```diff
@@ -0,0 +1,13 @@
+services:
+    terminal42.autoregistration.hook_listener.registration:
+        class: Terminal42\AutoRegistrationBundle\HookListener\Registration
+        arguments:
+            - '@contao.security.frontend_user_provider'
+            - '@security.token_storage'
+            - '@database_connection'
+            - '@monolog.logger.contao'
```

Comment:
Use `@logger` only and tag the logger to create a channel for this bundle:
```
    - { name: 'monolog.logger', channel: 'terminal42_autoregistration' }
```

Reply:
This was something new to me.